src/spell/modules/spell/spell/src/script.rs

```diff
@@ -20,21 +19,23 @@ fn check_env() {
 
 #[marine]
 pub fn set_script_source_to_file(script: String) -> UnitValue {
-    if !is_by_creator() {
-        return UnitValue::error("Only owner of the service can set the script");
+    if !is_by_creator() && !is_by_spell(&marine_rs_sdk::get_call_parameters()) {
```

Comment:
Anyone can set particle id in `spell_%service_id%_%count%` format. 
I think we should rely only on the service creator. Also currently, only workers own spells